### PR TITLE
geforce-now: Add nightly

### PR DESCRIPTION
**Pull request by bepoli**
Nightly, it autoupdates itself.
User's config is stored in `%localappdata%/NVIDIA Corporation`, so it's not 100% portable.

### CODE REVIEW

File: bucket/geforcenow.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "nightly",
+    "description": "NVIDIA’s cloud-based game streaming service",
+    "homepage": "https://www.nvidia.com/en-us/geforce-now/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.nvidia.com/en-gb/about-nvidia/nvidia-geforce-now-terms-of-use/"
```
Comment (Ash258) [inline bucket/geforcenow.json:7]:
```suggestion
        "url": "https://www.nvidia.com/en-us/geforce-now/terms-of-use/"
```

File: bucket/geforcenow.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    "url": "https://download.nvidia.com/gfnpc/GeForceNOW-release.exe#/dl.7z",
+    "extract_dir": "GeforceNOW",
+    "bin": "CEF\\GeForceNOW.exe",
+    "shortcuts": [
+        [
+        "CEF\\GeForceNOW.exe",
+        "GeForceNOW"
```
Comment (Ash258) [inline bucket/geforcenow.json:15]:
```suggestion
            "CEF\\GeForceNOW.exe",
            "GeForce NOW"
```

File: bucket/geforcenow.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "nightly",
+    "description": "NVIDIA’s cloud-based game streaming service",
+    "homepage": "https://www.nvidia.com/en-us/geforce-now/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.nvidia.com/en-gb/about-nvidia/nvidia-geforce-now-terms-of-use/"
+    },
+    "url": "https://download.nvidia.com/gfnpc/GeForceNOW-release.exe#/dl.7z",
```
Comment (Ash258) [inline bucket/geforcenow.json:9]: A

```suggestion
    "architecture": {
        "64bit": {
            "url": "https://download.nvidia.com/gfnpc/GeForceNOW-release.exe#/dl.7z"
        }
    },
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
`git mv bucket/geforcenow.json bucket/geforce-now.json

**Review by Ash258 — COMMENTED** [review]
Manifest installation and application is start is working correctly, but I cannot verify full application functionality

**Comment by benplm** [comment]
> Manifest installation and application is start is working correctly, but I cannot verify full application functionality

Do I need to provide some proof, like a screenshot of the working application (while streaming), or somebody else has to review this?
Sorry for the naive question.

File: bucket/geforce-now.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "extract_dir": "GeforceNOW",
+    "bin": "CEF\\GeForceNOW.exe",
+    "shortcuts": [
+        [
+            "CEF\\GeForceNOW.exe",
+            "GeForce NOW"
+        ]
+    ]
+}
```
Comment (pratikpc) [inline bucket/geforce-now.json:22]: Can you add autoupdate support?

Comment (issaclin32) [reply bucket/geforce-now.json:22]: I cannot find a trackable update log. Will try to add `autoupdate` via *UpdateTracker*.

**Comment by niheaven** [comment]
Version number may be found in https://www.nvidia.com/en-us/geforce-now/release-highlights/

**Comment by niheaven** [comment]
```json
"checkver": {
    "url": "https://www.nvidia.com/en-us/geforce-now/release-highlights/",
    "regex": "GeForce NOW ([\\d.]+)"
}
```

**Comment by issaclin32** [comment]
> Version number may be found in https://www.nvidia.com/en-us/geforce-now/release-highlights/

@niheaven Thanks for the info.
I thought about that one, but the "build version" is not on the page.
This will lead to **frequent hash check errors** in the future.

I am planning to use our new [UpdateTracker](https://github.com/ScoopInstaller/UpdateTracker) to download "first 8 bytes" to check if the file was changed, and track the full version number **by downloading the file**.

geforce-now2
 
geforce-now

**Comment by niheaven** [comment]
Is it a rare case that increase build version within same minor version? If it is, then manually redownload and hash the package may be simpler.

**Review by jasongodev — COMMENTED** [review]
I recommend closing this pull request if autoupdate feature is impossible to be done. The nature of this app is also not a perfect fit for the Extras bucket to be included.

**Pull request closed without merging**